libcontainer/cgroups/systemd/v1.go

```diff
@@ -233,48 +240,25 @@ func (m *legacyManager) Path(subsys string) string {
 	return m.paths[subsys]
 }
 
-func join(c *configs.Cgroup, subsystem string, pid int) (string, error) {
```

Comment:
the body of this function (mobulo `getSubsystemPath`) was inlined into `joinCgroups` (see below)